src/Factories/TestCaseFactory.php

```diff
@@ -132,10 +132,11 @@ public function build(TestSuite $testSuite): array
         $proxies     = $this->proxies;
         $factoryTest = $this->test;
 
-        $test = function () use ($chains, $proxies, $factoryTest): void {
+        $test = function () use ($chains, $proxies, $factoryTest) {
```

Comment:
The return type should be `mixed`. If there is phpstan issues I will fix them in my side later.

Reply:
Like so?
![image](https://user-images.githubusercontent.com/13331388/85172032-7fe4be00-b270-11ea-989e-4762c3e03ee0.png)

Reply:
If `phpstan` do not complains, yes.